APPLICATION_PROGRESS.md

```diff
@@ -338,16 +338,18 @@ A user should be able to see the campaigns that they have pledged towards
 <details>
 <summary>Name Registry</summary>
 
-<h3>Contracts</h3>
+<h3>Contracts ❌</h3>
+
+- str[8] is used as a temporary workaround due to dynamic strings not being available in storage
```

Comment:
It may be useful to link the StorageString issue https://github.com/FuelLabs/sway-libs/issues/40